webpages/settings/index.js

```diff
@@ -787,8 +787,7 @@ const vue = (window.vue = new Vue({
     },
     setTheme(mode) {
       chrome.storage.sync.get(["globalTheme"], function (r) {
-        let rr = true; //true = light, false = dark
-        rr = mode;
+        let rr = mode ?? true;
```

Comment:
lol `??` is chrome 80+, we almost cant use it

Reply:
Oof